nss-cohort-36/bangazon-workforce-mgt-illin-illusionists
# Rb computer delete

# Description
User can delete a computer from their inventory ONLY if it has never been assigned.

Closes #12 
## Type of change

### Setup 

Ensure you are in your `/bangazon-workforce-mgt-illin-illusionists/hrapp/` directory then setup with the following commands

```bash
git fetch --all
git checkout rb-computer-delete
python manage.py runserver
```

### Testing Steps
1. Go to the `computers` list
2. Select a computer link to go to the `details` view
3. Make sure you have a `Delete` button available
4. When you click the `Delete` button, a dialog should pop-up asking you to confirm deletion of that item number
5. When you click `Cancel` the dialog should close and nothing else happens
6. When you click `Delete` you will be returned to the `computers` list and the computer MIGHT be deleted
7. Try this on each computer. You will see that only computers that have never been assigned to an employee will be removed.
NOTE: this matches the requirements of the ticket, and I fully acknowledge that it is not intuitive to allow the user to even have a dialog box to delete if that computer cannot actually be deleted. #MaliciousCompliance 😄 


## Checklist:
